meeting-minutes/2023-04-25.md

```diff
@@ -0,0 +1,244 @@
+
+Jessica: I'm very positive, especially after this meeting. there's obviously been a lot of convo on the concrete syntax, I'm very invested in, I did a lot of work in the SocialWG and it didn't necessarily have interop and over the course of the group was difficult to work through. I'm interested in spec work in terms of getting implementations to talk with each other, getting new implementations, creating documentation and
+
+Jessica: technologies that further this space.  I'm very excited about that, and certainly excited to work with Mark and everyone at Agoric to work on this.  I think we have a good shot of coming to consensus on this
+
+markm: I agree with the general positivity. I think it's very interesting how the issues have... as we've broken things into layers that's been clarifying for me too, made my vision sharper. until recently I kept talking about 3 layers, all of which were about data, and the nice thing in
+Jessica's document is it's focusing on messaging in terms of capabilities, and that's really nice, we've got our data and slots arrays and things
+
+markm: have to be translated from clist context to clist context.  I think that's going to be very very challenging, we *need* to have blockchains be able to participate in the fabric, that's of course been central to what Agoric has done.  IBC *is* the netlayer for talking to chains, and it's been very designed to meet these needs, and doing 3 party handoffs between combinations of chains and non-chains, we'll likely find it quite
+
+markm: challenging, and it's necessary to bite this off.  One thing I noticed about the document I read, it's more like old E, making assumptions that endpoints can keep secrets.  weird thing about blockchain is that the endpoints can't keep secrets.  One final note about data agreement is that we're *really* stuck on the abstract syntax, so things that are still minor issues are gating issues, there's two levels of tagging, distinction
+
+markm: between record / string / symbol, and then there's the Tag, and we can't bridge to an encoding that takes some of the things problematic to you like null and just tosses it into the tag, because then it's not "the tag".  if we did that it's an extension of the first level tagging, we'd still encode our ? as the top level tagging, because the tag has to be exaclty the tag
+
+isd: also to echo the general positivity, I think... the kind of approach I am taking to this is kind of a one-issue-at-a-time thing.  we'd been makign good progress on the data model thing, and I think I'll try to comment, how does everyone like this particular form, because one of the things I found... it's a bit difficult I found to navigate the comments in the agoric source to understand the data model, so I might need to try to
```

Comment:
```suggestion
isd: also to echo the general positivity, I think... the kind of approach I am taking to this is kind of a one-issue-at-a-time thing.  we'd been making good progress on the data model thing, and I think I'll try to comment, how does everyone like this particular form, because one of the things I found... it's a bit difficult I found to navigate the comments in the agoric source to understand the data model, so I might need to try to
```